callisto/delivery/api.py

```diff
@@ -20,33 +43,25 @@ class DeliveryApi(object):
         DeliveryApi maps to
     '''
 
-    def __init__(self):
-        self.set_api_class()
+    api_env_variable = 'CALLISTO_NOTIFICATION_API'
+    default_classpath = 'callisto.delivery.api.AbstractNotification'
 
     def __getattr__(self, attr):
         if attr in AbstractNotification.__dict__.keys():
```

Comment:
yeah, I didn't bother with enforcing the API on MatchingApi because there's only one method that's really intended to be overridden